From c0a0d379296f562995a790a52d4160092906f067 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Wed, 6 Jun 2018 00:28:42 +0200
Subject: [PATCH] Makefile: align for oe-cross-build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 Makefile | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/Makefile b/Makefile
index 951ddb3..5f57a34 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,3 @@
-BIN_PATH                = /usr/local/bin
-LIB_PATH                = /usr/local/lib
-DOC_PATH                = /usr/local/share/doc
 LADSPA_PATH             = $(LIB_PATH)/ladspa
 LV2_PATH                = $(LIB_PATH)/lv2
 VST_PATH                = $(LIB_PATH)/vst
@@ -26,8 +23,6 @@ export RESDIR           = ${CURDIR}/res
 export RELEASE          = ${CURDIR}/.release
 export BUILDDIR         = $(OBJDIR)
 
-# Dependencies
-export VST_SDK          = /home/sadko/eclipse/lsp-plugins-vst3sdk
 
 # Includes
 INC_FLAGS               = -I"${CURDIR}/include"
@@ -55,14 +50,9 @@ endif
 export BASEDIR          = ${CURDIR}
 export INCLUDE          = ${INC_FLAGS}
 export MAKE_OPTS        = -s
-export CFLAGS           = $(CC_ARCH) -std=c++98 -fPIC -fno-exceptions -fno-asynchronous-unwind-tables -Wall -pthread -pipe -fno-rtti $(CC_FLAGS) -DLSP_MAIN_VERSION=\"$(VERSION)\"
-export CC               = g++
+export CC               = $(CXX)
 export PHP              = php
-export LD               = ld
-export LDFLAGS          = $(LD_ARCH) 
-export SO_FLAGS         = $(CC_ARCH) -Wl,-rpath,$(LD_PATH) -shared -Llibrary -lc -lm -fPIC -lpthread
-export MERGE_FLAGS      = $(LD_ARCH) -r
-export EXE_FLAGS        = $(CC_ARCH) -Wl,-rpath,$(LD_PATH) -lm -fPIC -pthread
+export LD               = $(CXX)
 
 # Objects
 export OBJ_CORE         = $(OBJDIR)/core.o
-- 
2.14.3

